packages/fast-components-react-msft/src/metatext/metatext.props.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { IManagedClasses, IMetatextClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+export enum MetatextTag {
+    p = "p",
+    span = "span"
+}
+
+export interface IMetatextHandledProps {
+    /**
+     * The metatext content
+     */
+    children?: React.ReactNode | React.ReactNode[];
+
+    /**
+     * The metatext tag
+     */
+    tag: MetatextTag;
```

Comment:
this would be optional if you utilized defaultProps

Reply:
This should be optional anyway due to the current getter setup. This is just a mistake.